run_docker

```diff
@@ -69,8 +71,19 @@ PYODIDE_DOCKER_PORT=${PYODIDE_DOCKER_PORT:-"8000"}
 PYODIDE_SYSTEM_PORT=${PYODIDE_SYSTEM_PORT:-${DEFAULT_PYODIDE_SYSTEM_PORT}}
 PYODIDE_DOCKER_IMAGE=${PYODIDE_DOCKER_IMAGE:-${DEFAULT_PYODIDE_DOCKER_IMAGE}}
 
+# in case the port is not a number, do not bind the port
+case $DEFAULT_PYODIDE_SYSTEM_PORT in
+  ''|*[!0-9]*) # contains a non-digit character, therefore it is not a number
+  echo "WARNING: port binding disabled (received '$DEFAULT_PYODIDE_SYSTEM_PORT')"
```

Comment:
Probably would be good to have a separate "none" case which doesn't display this message.